app-portage/eix/eix-0.32.1.ebuild

```diff
@@ -26,31 +31,39 @@ DEPEND="${BOTHDEPEND}
 	nls? ( sys-devel/gettext )"
 
 pkg_setup() {
-	case " ${REPLACING_VERSIONS}" in
-	*\ 0.[0-9].*|*\ 0.1[0-9].*|*\ 0.2[0-4].*|*\ 0.25.0*)
-		local eixcache="${EROOT}/var/cache/${PN}"
-		test -f "${eixcache}" && rm -f -- "${eixcache}";;
-	esac
+	# remove stale cache file to prevent collisions
+	local old_cache=${EROOT%/}/var/cache/${PN}
+	if [[ -f ${old_cache} ]]; then
+		rm "${old_cache}" || die
+	fi
 }
 
 src_prepare() {
-	sed -i -e "s'/'${EPREFIX}/'" -- "${S}"/tmpfiles.d/eix.conf || die
-	eapply_user
+	default
+	sed -i -e "s,/,${EPREFIX}/," tmpfiles.d/eix.conf || die
```

Comment:
(By the way, `:` would be very impractical in `${EPREFIX}` because `/my:eprefix/usr/bin` (or `/my\:eprefix/usr/bin` etc.) does not work in `${PATH}`, while `/my,eprefix/usr/bin` works in `${PATH}`.)